locales/it/index.yml

```diff
@@ -1296,6 +1296,11 @@ wallet:
       success:
         title: "Grazie, l’operazione è stata eseguita con successo!"
         description: "Abbiamo inviato una ricevuta al tuo indirizzo email. Puoi trovarla anche nel tuo Portafoglio."
+  abortWebView:
+    title: Vuoi interrompere l'aggiunta della carta di credito in corso?
+    confirm: Annulla l'aggiunta
+    cancel: Continua con l'aggiunta
+    success: L'aggiunta della carta di credito è stata annullata correttamente
```

Comment:
@debiff @Undermaken this is quite an interesting use case. If you abort the flow, it's very likely you're having issues with the 3DS codes. Maybe we can suggest users to get in touch with their bank. I'll create a story ;)